### ISSUE TITLE
is it possible to search items by one atribute like an id and get everething with the same atribute?

### ISSUE THREAD

**Issue by gabrielMoran23** [vgrem/phpSPO#172]
i have an attribute called candidate_id, is it possible to search files with this atribute, i cannot find this method.

**Comment by micah500** [comment]
Did you have any luck with this? I'm in the same boat...

**Comment by gabrielMoran23** [comment]
Yes, if you don't have the solution let me show you my code tomorrow :D

**Comment by micah500** [comment]
Hi @gabrielMoran23, I still haven't figured it out. Any code you can share would be greatly appreciated. I'm basically trying to search a List and get results WHERE 'Title'='myTitle'.
Again, any help is so very appreciated!

**Comment by gabrielMoran23** [comment]
Wait me I forgot it but in a 1 hour ill show u my code

**Comment by gabrielMoran23** [comment]
Hey I hope this helps you n.n

**Comment by micah500** [comment]
Hey @gabrielMoran23, thanks heaps for giving it a try! Could you please check that the code example made it to the GitHub website? I don't think it's coming through above. Thanks again! Micah

**Comment by gabrielMoran23** [comment]
Let me check :)
